read_env_props.py

```diff
@@ -0,0 +1,21 @@
+import os
+import logging
+from typing import DefaultDict
+logger = logging.getLogger(__name__)
+
+class ReadEnvProps:
+    def __init__(self):
+        self.env_props: DefaultDict = {}
+        self._read_env_props()
+
+    def _read_env_props(self) -> None:
+        logger.debug("Reading environment properties")
+        self.env_props['sheet_id'] = os.environ.get('IPE_SHEET_ID', '1fvYhtxlu3pa8vNBjOVzp1hwNBsv823mTD3uP6l5_WuI')
```

Comment:
I don't think any ID needs to be hardcoded here.

Do you really need this at all? Seems like you could just read from the environ when you need it later rather than have this code move from one dict to another.

Reply:
That's fine I guess, I'd leave the default values out of here though and just pull it from the environment.

Here's a couple ways you can source the .env before running the command.
https://stackoverflow.com/a/40181231/3708872